src/main/java/io/vertx/core/json/pointer/JsonPointer.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) 2011-2017 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.json.pointer;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.Nullable;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.json.pointer.impl.JsonPointerImpl;
+
+import java.net.URI;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+/**
+ * Implementation of <a href="https://tools.ietf.org/html/rfc6901">RFC6901 Json Pointers</a>.
+ *
+ * @author Francesco Guardiani <a href="https://slinkydeveloper.github.io/">@slinkydeveloper</a>
+ */
+@VertxGen
+public interface JsonPointer {
+
+  /**
+   * Return true if the pointer is a root pointer
+   *
+   * @return
+   */
+  boolean isRootPointer();
+
+  /**
+   * Return true if the pointer is local (URI with only fragment)
```

Comment:
Instead :

"@return {@code true} if the pointer is local (URI with only fragment)"